app/controllers/result_filters/subject_controller.rb

```diff
@@ -16,7 +16,7 @@ def start
     end
 
     def create
-      if params[:subjects].blank? && (params[:senCourses].blank? || params[:senCourses] == 'false')
+      if params[:subject_codes].blank? && (params[:senCourses].blank? || params[:senCourses] == 'false')
```

Comment:
Very minor nit pick but wondering if we should be using the `subject_code` rather than `subject_codes` so we have some thing like:
`/results?l=2&subject_code%5B%5D=00&subject_code%5B%5D=01&subject_code%5B%5D=02`

There is also a larger question about whether Find params should be hyphenated rather than snake-cased

Reply:
So the problem is though that it munges it into an array in the controllers so it's always going to be dramatically incorrect somewhere. Feels like this this might be the lesser evil otherwise you're iterating over `subject_code` wdyt?

Reply:
Ah right - np